modules/util/css-builder/src/test/java/com/liferay/css/builder/maven/BuildCSSMojoTest.java

```diff
@@ -0,0 +1,177 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.css.builder.maven;
+
+import com.liferay.maven.executor.MavenExecutor;
+import com.liferay.portal.kernel.util.StringUtil;
+
+import java.io.File;
+import java.io.IOException;
+
+import java.nio.file.DirectoryStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+/**
+ * @author Christopher Bryan Boyd
+ */
+public class BuildCSSMojoTest {
+
+	@ClassRule
+	public static final MavenExecutor mavenExecutor = new MavenExecutor();
+
+	@BeforeClass
+	public static void setUpClass() throws IOException {
+		_cssBuilderVersion = System.getProperty("cssbuilder.version");
```

Comment:
These could simply system property variables can simply be moved into static fields in teh class.  No need to create them in the setupclass method.